test_api_data.py

```diff
@@ -0,0 +1,49 @@
+import api_data as api
+
+from mock import Mock
+import pytest
+from urllib.request import urlopen
+
+test_data = '''
+{	"data": [
+        {
+            "Modified": "2008-11-15T00:00:00",
+            "Published": "2007-02-19T21:28:00",
+            "access": {
+                "authentication": "NONE",
+                "complexity": "MEDIUM",
+                "vector": "NETWORK"
+            },
+            "cvss": 4.3,
+            "cvss-time": "2007-02-20T14:55:00",
+            "id": "CVE-2007-1004",
+            "impact": {
+                "availability": "NONE",
+                "confidentiality": "NONE",
+                "integrity": "PARTIAL"
+            },
+            "reason": "Link",
+            "references": [
+                "http://securityreason.com/securityalert/2264",
+                "http://www.securityfocus.com/archive/1/archive/1/460369/100/0/threaded",
+                "http://www.securityfocus.com/archive/1/archive/1/460412/100/0/threaded",
+                "http://www.securityfocus.com/archive/1/archive/1/460617/100/0/threaded",
+                "http://www.securityfocus.com/bid/22601",
+                "http://xforce.iss.net/xforce/xfdb/32580"
+            ],
+            "summary": "Mozilla Firefox might allow remote",
+            "vulnerable_configuration": [
+                "cpe:2.3:a:mozilla:firefox:2.0:rc3"
+            ],
+            "vulnerable_configuration_cpe_2_2": [
+                "cpe:/a:mozilla:firefox:2.0:rc3"
+            ]}]}
+'''
+
+def test_output_cve_id():
+    #BUG: 
```

Comment:
@kartiksibal the think I am not comfy with is the `#BUG` comment which is meaningless... either you explain in a full narrative comment what the bug is or you remove the  `#BUG` line ;)